packages/components/pie-icon-button/test/visual/pie-icon-button.spec.ts

```diff
@@ -1,8 +1,17 @@
 import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
+import type {
+    PropObject, WebComponentPropValues, WebComponentTestInput,
```

Comment:
If we hate the fact that all of these imports have to be split, I wonder if we could have a test helper module that imports all of these from their separate locations and then re-exports them to make it easier to consume

Reply:
I think that's what leads to the directory imports error. Can you try and test this locally please?